docs/_writing-tests/testdriver.md

```diff
@@ -48,4 +48,34 @@ between the function being called and the promise settling.
 To send special keys, one must send the respective key's codepoint. Since this uses the WebDriver protocol, you can find a [list for code points to special keys in the spec](https://w3c.github.io/webdriver/webdriver-spec.html#keyboard-actions).
 For example, to send the tab key you would send "\uE004".
 
+### `test_driver.pointer_action_sequence(actions)`
+ - `actions` <[Array]<[Object]>> an array of Action objects`
+  - `action` <[Object]> A single action
+   - `id` The ID of the pointer to use. If one with this id does not exist a new one will be used
+   - `pointerType` The type of pointer to use, an enum of `mouse`, `touch` and `pen`.
+   - `subtype` The type of action to perform, an enum of `pointerUp`, `pointerMove`, `pointerDown` and `pause`
+   - `target` Either a DOM Element object or an Object with exactly two integer attrbutes `x` and `y`.
```

Comment:
I wonder if we should allow the float coordinates as the events may have float coordinates in high dpi environments.

Reply:
The webdriver spec has an error thrown if these aren't integers, so for now I think we should keep this with that spec; however, this is a good point and maybe the spec should change.